src/components/Calculator.js

```diff
@@ -3,45 +3,56 @@ import React, { Component } from 'react';
 import './Calculator.css';
 import Button from './Button';
 
+import calculate from './logic/calculate';
+
```

Comment:
- [ ] The Calculator component should import the `calculate.js & operate.js` files from the logic directory but currently, you imported `calculate` only kindly fix that
- [ ] I recommend that you move out `logic` folder outside the component

Reply:
Hello Lydia,
Thanks for the review. The operate class is being used in the calculate.js file which has the only function that I need to apply to the onclick events. If I import the operate, then I will get the "no-unused-vars" linter error.